frontend/src/app/patients/personSchema.ts

```diff
@@ -237,7 +236,7 @@ const updateFieldSchemata: (
   city: yup
     .string()
     .max(MAX_LENGTH, t("patient.form.errors.fieldLength", undefined))
-    .nullable(),
+    .required(t("patient.form.errors.city", undefined)),
```

Comment:
It looks like some implementation details may leak through in some situations:

<img width="290" alt="Screen Shot 2022-12-05 at 1 42 34 PM" src="https://user-images.githubusercontent.com/27730981/205717176-bf135b4d-e10c-411d-a504-383ab7fb3ec7.png">

To replicate, you can simply go to the patient self-registration form and click on the language toggler in the top-right.

Reply:
Is there any chance you were doing something more than what you're describing?  I'm not seeing this behavior locally.  Not seeing any errors when I toggle language for an empty form.  And when I toggle language for a filled out form with the city missing (manually deleted after fake filler) - I'm seeing the expected error message.

Reply:
@DanielSass I think it needs to be fixed here so null is not an option
https://github.com/CDCgov/prime-simplereport/blob/8dc5c6c84583dda8cbf7da58a1c15cda9dfbb94a/frontend/src/app/Settings/types.d.ts#L37

Reply:
second option did it, thanks Merethe